fix 'error: format not a string literal and no format arguments [-Werror=format-security]'
--- a/engines/nancy/action/puzzle/sortpuzzle.cpp
+++ b/engines/nancy/action/puzzle/sortpuzzle.cpp
@@ -293,7 +293,7 @@
 				const Cell &cur = _current[r][c];
 				line += Common::String::format("%d ", cur.value);
 			}
-			debug(line.c_str());
+			debug("%s", line.c_str());
 		}
 		debug("-----");
 	}
